packages/record/src/consentVideo.spec.ts

```diff
@@ -179,10 +182,12 @@ test("recordButton", async () => {
   const jsPsych = initJsPsych();
   const plugin = new VideoConsentPlugin(jsPsych);
   const display = document.createElement("div");
+  const trial = { locale: "en-us" } as unknown as TrialType<PluginInfo>;
+
+  display.innerHTML = chsTemplates.consentVideo(trial);
 
   display.innerHTML =
-    Handlebars.compile(consentVideoTrial)({}) +
-    Handlebars.compile(recordFeed)({});
+    chsTemplates.consentVideo(trial) + Handlebars.compile(recordFeed)({});
```

Comment:
Ah. So we should keep using `Handlebars.compile` if the template has no translations?

Reply:
That seems correct for now.  I am not sure how this is all going to shake out. Do you have a preference in this space?

Reply:
No preference!